spec/views/councillor_contributions/new_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+feature "Contributing new councillors for an authority" do
```

Comment:
Nice work moving this over. This spec is still really a feature spec [using Capybarra](https://github.com/teamcapybara/capybara#using-capybara-with-rspec) methods to interact with the app like visit. We just want to test what renders in the view file here when a specific Authority is assigned. We can use the rspec view spec methods for this. Here's the rspec documentation for that https://relishapp.com/rspec/rspec-rails/docs/view-specs/view-spec .